Lib/fontmake/font_project.py

```diff
@@ -172,6 +180,9 @@ def build_master_ufos(
         except Exception as e:
             raise FontmakeError("Loading Glyphs file failed", glyphs_path) from e
 
+        if font.customParameters["Enforce Compatibility Check"]:
+            self.check_compatibility = True
```

Comment:
hm the FontProject object is stateful.. It's fine if the CLI option is set on the object self.check_compatibility attribute, since the FontProject is made there and thrown away after main() is done. But the fact that one particular GSFont has that customparameter should not change the state of the FontProject's check_compatibility option (which would apply for subsequent runs as well). Could you find a way to trickle that option down to where it's used without modifying self?